sdk/appconfiguration/azure-appconfiguration/tests/recordings/test_azure_configuration_client.test_add_configuration_setting.yaml

```diff
@@ -22,13 +22,13 @@ interactions:
       x-ms-content-sha256:
       - w631Ltx6sQIAsaaqTsE1t6u383QdMo18I00bnNGSih4=
       x-ms-date:
```

Comment:
Can we revert all these recording changes?

Modifying transport should not impact recordings.

Reply:
I was a little surprised by it, but the recordings are actually missing a good number of fields when recorded with `AsyncioRequestsTransport` vs. with `AioHttpTransport`. Specifically, everything was failing in playback when using `AioHttpTransport` because recordings that use `AsyncioRequestsTransport` don't include a `url` field in responses -- `vcrpy` breaks when using `aiohttp` if this field is missing.

It seems that recordings aren't transport-independent with `vcrpy`, even though one would hope they would be 🙃